### PR TITLE
Include page number in title

### PR DESCRIPTION
**Pull request by mpchadwick**
Per [Moz's article "Pagination: Best Practices for SEO & User Experience"](https://moz.com/blog/pagination-best-practices-for-seo-user-experience) it's not ideal to use the same title with pagination. This change follows Moz's suggestion and appends "Page X of Y for" to the beginning of the `<title>` when on page 2 of deeper.

### CODE REVIEW

**Comment by DirtyF** [comment]
@jekyllbot: merge +minor

**Comment by DirtyF** [comment]
Thanks @mpchadwick 😃

File: lib/jekyll-seo-tag/drop.rb [hunk 1]
```diff
@@ -53,6 +53,12 @@ def title
             page_title || site_title
           end
         end
+
+        if page_number
+          return page_number + @title
+        end
```
Comment (pathawks) [inline lib/jekyll-seo-tag/drop.rb]: Is it intentional that we are not memoizing this?

**Comment by chesterhow** [comment]
Is there a way to disable this feature?

**Pull request merged**